homeassistant/components/lektrico/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""The Lektrico Charging Station integration."""
+from __future__ import annotations
+
+import logging
+
+from lektricowifi import lektricowifi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DATA_LEKTRICO_CLIENT, DOMAIN
+
+# from homeassistant.components.lektrico.sensor import LektricoSensor
+
+
+# List the platforms that you want to support.
+PLATFORMS = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Lektrico Charging Station from a config entry."""
+    session = async_get_clientsession(hass)
+    charger = lektricowifi.Charger(
+        entry.data[CONF_HOST],
+        session=session,
+    )
+
+    # Ensure we can connect to it
+    try:
+        await charger.charger_info()
+    except lektricowifi.ChargerConnectionError as exception:
+        logging.getLogger(__name__).debug("Unable to connect: %s", exception)
+        raise ConfigEntryNotReady from exception
+
+    hass.data.setdefault(DOMAIN, {})
+    hass.data[DOMAIN][entry.entry_id] = {DATA_LEKTRICO_CLIENT: charger}
```

Comment:
Shorter:

```suggestion
    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {DATA_LEKTRICO_CLIENT: charger}
```

Additionally, any reason why you are using a dictionary here? Could it just this instead?

```py
    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = charger
```